lib/segment/open.c

```diff
@@ -21,10 +21,28 @@
 /**
  * \brief Initialize segment structure and open segment file.
  *
- * Initializes the <b>seg</b> structure and prepares a temporary file. 
- * This fn is a wrapper for Segment_format() and Segment_init()
+ * Initializes the *seg* structure and prepares a temporary file.
  *
- * <b>Note:</b> The file with name fname will be created anew.
+ * This function is a wrapper for Segment_format_nofill(),
+ * Segment_init(), all-in-memory mode initialization, and temporary
+ * file handling.
+ *
+ * The values are not guaranteed to be initialized to zero or NULL.
```

Comment:
not values, but bytes, they are indeed initialized to zero because of lseek and calloc